app/src/main/res/layout/activity_main.xml

```diff
@@ -88,12 +88,23 @@
                         android:background="@drawable/searchview_rounded_bg"
                         android:gravity="center"
                         android:hint="Search..."
-                        android:textColorHint="@color/black"
                         android:paddingStart="5dp"
                         android:scrollHorizontally="true"
                         android:textAlignment="viewStart"
                         android:textAllCaps="false"
-                        android:textSize="15sp" />
+                        android:textColorHint="@color/black"
+                        android:textSize="14sp" />
+
+                    <ImageButton
+                        android:id="@+id/buttonClear"
+                        android:layout_width="wrap_content"
+                        android:layout_height="20dp"
+                        android:layout_marginStart="245dp"
+                        android:layout_marginTop="-27dp"
+                        android:background="#000000"
```

Comment:
Use the color black constant, it is at colors.xml

Reply:
Done.